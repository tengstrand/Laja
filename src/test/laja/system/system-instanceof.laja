
#set (number = loadClass("java.lang.Number"))

#write "target/test-output/system-instanceof.out"
  #if (instanceof(1, number))
    1 is instanceof java.lang.Number
  #end
  #if (instanceof(number, 1))
    This is not true!
  #end
  #if (instanceof(1, 2))
    1 is instanceof 2
  #end
#end
