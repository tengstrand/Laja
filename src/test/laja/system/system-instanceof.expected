    1 is instanceof java.lang.Number
    1 is instanceof 2
